docs/csharp/programming-guide/concepts/linq/linq-to-xml-overview.md

```diff
@@ -5,6 +5,10 @@ ms.assetid: 716b94d3-0091-4de1-8e05-41bc069fa9dd
 ---
 # LINQ to XML Overview (C#)
 
+LINQ to XML provides an in-memory XML programming interface that leverages the .NET Language-Integrated Query (LINQ) Framework. LINQ to XML uses the latest .NET Framework language capabilities and is comparable to an updated, redesigned Document Object Model (DOM) XML programming interface. 
```

Comment:
.NET Framework language capabilities --> .NET capabilities? .NET LINQ Framework capabilities?

Reply:
And, "the latest" is not really relevant anymore either.